src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseHelper.cs

```diff
@@ -714,6 +714,14 @@ internal static void AddSuggestionsForGlobals(IntellisenseData.IntellisenseData
 
                 AddSuggestion(intellisenseData, funcNamespace.Name, SuggestionKind.Global, SuggestionIconKind.Other, DType.Unknown, requiresSuggestionEscaping: true);
             }
+
+            if (intellisenseData.Binding.NameResolver is IGlobalSymbolNameResolver nr2 && nr2.GlobalSymbols != null)
+            {
+                foreach (var symbol in nr2.GlobalSymbols.Where(symbol => IsMatch(symbol.Key, intellisenseData.MatchingStr)))
+                {
+                    CheckAndAddSuggestion(new IntellisenseSuggestion(new UIString(symbol.Key), SuggestionKind.Global, SuggestionIconKind.Other, symbol.Value.Type, -1, symbol.Value.DisplayName, null, null), intellisenseData.Suggestions);
```

Comment:
`symbol.Value.DisplayName` here can be cleaned up, it's always null after the last iteration

Reply:
We should be safe then